handout/features/spread_and_rest.md

```diff
@@ -1,14 +1,14 @@
-# Operators Spread and Rest
+# Spread Syntax (Spread Element) and Rest parameters
 
-A Spread operator allows in-place expansion of an expression for the following cases:
+A Spread syntax allows in-place expansion of an expression for the following cases:
```

Comment:
To add more confusion to the mix. If we refer always to "Rest parameters" then what term should we use to refer to this?

```js
[a, b, ...rest] = [1, 2, 3, 4, 5];
```